pages/common/scp.md

```diff
@@ -1,30 +1,28 @@
 # scp
 
-> Copies files between hosts on a network.
-> Works over a secure connection (SSH).
+> Secure copy.
+> Copy files between hosts using Secure Copy Protocol over SSH.
 
-- Upload a file, or upload and rename a file:
+- Copy a loocal file to a remote host:
 
-`scp {{/local/file.txt}} {{10.0.0.1}}:{{/remote/path/}}`
-`scp {{/local/file.txt}} {{10.0.0.1}}:{{/remote/path/newname.txt}}`
+`scp {{local_file}} {{remote_host}}:{{/path/remote_file}}`
 
-- Download a file:
+- Copy a file from a remote host to a local folder:
 
-`scp {{10.0.0.1}}:{{/remote/path/file.txt}} {{/local/folder}}`
+`scp {{remote_host}}:{{/path/remote_file}} {{/path/local_dir}}`
 
-- Upload or download a directory:
+- Recursively copy the contents of a directory on a remote host to a local directory:
 
-`scp -r {{/local/folder}} {{10.0.0.1}}:{{/remote/path/}}`
-`scp -r {{10.0.0.1}}:{{/remote/path}} {{/local/folder}}`
+`scp -r {{remote_host}}:{{/path/remote_dir}} {{/path/local_dir}}`
 
-- Specify username on host:
+- Copies a file between two remote hosts transferring through the local host:
```

Comment:
Prefer infinitive tense, "copy", over "copies".

I guess this replaces the copy from one host to another example below? This adds `-3`, I looked it up and it does what you describe. Then what's the default? Since the previous author left out the parameter.